src/StorageSync/StorageSync/help/Remove-AzStorageSyncCloudEndpoint.md

```diff
@@ -106,7 +106,7 @@ Accept wildcard characters: False
 ```
 
 ### -Name
-Name of the CloudEndpoint.
+Name of the CloudEndpoint. Name of the CloudEndpoint. The name is a GUID, not the friendly name that's displayed in the portal. To get the CloudEndpointName, use the Get-AzStorageSyncCloudEndpoint cmdlet.
```

Comment:
We cannot state that the name is always a GUID, because it's not the case. When creating a cloud endpoint through the portal, it sets the name to be a GUID, but when cloud endpoints are created through other clients (PowerShell, C# SDK, etc), then customers are free to set whatever name they want.

Here's what I suggest instead:
Name of the CloudEndpoint. To verify the cloud endpoint name, use the Get-AzStorageSyncCloudEndpoint cmdlet, and check the Name property of the returned object..

Reply:
I updated the description. Thanks!